index.js

```diff
@@ -0,0 +1,28 @@
+import BookList from './modules/books.js';
+
+import { DateTime } from './modules/date.js';
+
+import {
+  newBook,
+  showhide,
+} from './modules/elements.js';
+
+BookList.displayall();
+
+document.querySelector('.add').addEventListener('click', newBook);
+
+const timeDate = document.querySelector('.date');
+
+const dateAndTime = () => {
```

Comment:
- [ ] Kindly follow best ES6 practices, and create a module for the function `dateAndTime`. It will give your code a better structure and order